agenta-backend/agenta_backend/utils/common.py

```diff
@@ -1,22 +1,46 @@
+import logging
 from bson import ObjectId
 from odmantic import query
-from fastapi.responses import JSONResponse
-from typing import Dict, List, Union, Optional
+from fastapi.types import DecoratedCallable
+from fastapi import APIRouter as FastAPIRouter
 from agenta_backend.models.db_engine import DBEngine
+from typing import Dict, List, Union, Optional, Any, Callable
 from agenta_backend.models.db_models import (
     UserDB,
     AppVariantDB,
     OrganizationDB,
     AppDB,
     VariantBaseDB,
 )
-import logging
 
 engine = DBEngine().engine()
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.DEBUG)
 
 
+class APIRouter(FastAPIRouter):
+    def api_route(
```

Comment:
Can you add a docstring to this function to explain what it does in simple terms? Docstring format should be in Google Style.